{"levelname": "ERROR", "asctime": "2023-12-19T21:08:11.019Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 69, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "83cf80cb-0322-4be0-af52-ccc6429a9820", "collection_version_id": "ae65cba3-2559-4997-a78e-99dfe04e8946", "errors": ["error message"]}
